### PR TITLE
Donor can edit profile page

### PR DESCRIPTION
**Pull request by swardbladpaulo**
[Donor can edit information on Profile page](https://www.pivotaltracker.com/story/show/176626835)
### User Story
```
As a donor,
In order to display correct information
I want to be able to add and update my profile
```
## Changes proposed in this pull request:
* Creating a test for DonorCanEditProfilePage
*Creating a fixture for donor_can_edit_profie_page
*Creating a ProfileServices.js module
* Adding a updateProfileMessage: action.payload to rootReducer.js
* Change to use switch and route in app.jsx 

## What have we learned learned working on this feature:
* Semantic can cause issues
## Screenshots (Client)
Screenshot from 2021-02-02 14-07-36

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -4,20 +4,27 @@ import Footer from './components/Footer'
 import './index.css'
 import MainPageDisplay from './components/MainPageDisplay'
 import CreateFoodBagForm from './components/CreateFoodBagForm'
-import { useSelector } from 'react-redux'
+import { Switch, Route } from 'react-router-dom'
+import ProfileForm from './components/ProfileForm'
 
 const App = () => {
-  const currentUser = useSelector(state => state.currentUser)
+  // const currentUser = useSelector((state) => state.currentUser)
```
Comment (FlisAnn) [inline src/App.jsx]: Pls remove comment :)

File: src/App.jsx [hunk 2]
```diff
@@ -4,20 +4,27 @@ import Footer from './components/Footer'
...
+        </Switch>
         <Footer />
       </Suspense>
     </>
   )
 }
 
 export default App
+
+// {currentUser ? <CreateFoodBagForm /> : <MainPageDisplay />}
```
Comment (FlisAnn) [inline src/App.jsx]: Is this functionality removed now? Or refactored above? If so remove comment

**Review by FlisAnn — CHANGES_REQUESTED** [review]
Good Job, left two comments about code that was commented out

File: src/App.jsx (hunk 1 above)
Comment (swardbladpaulo) [reply src/App.jsx]: fixed

File: src/App.jsx (hunk 2 above)
Comment (swardbladpaulo) [reply src/App.jsx]: fixed

**Pull request merged**